lms/static/js/student_account/views/FormView.js

```diff
@@ -215,6 +215,11 @@
                     return data;
                 },
 
+                setUsaStateData: function(data){
+                    data.usa_states = $("#usa-regions").val();
```

Comment:
Here you change `data` by assigning value to `data.usa_states`, do you really need it? And if you do - there is no sense to return `data` because it has been already changed

Reply:
I really need it because i need send data from region field to back-end and write value from field to database.